bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java

```diff
@@ -304,11 +303,15 @@ public static JSONObject getJsonOfRequestParameters(SlingHttpServletRequest requ
      * @param linkHandler The link handler
```

Comment:
@jckautzmann - we should document the new style parameter

Reply:
@richardhand Addressed in https://github.com/adobe/aem-core-wcm-components/pull/1828/commits/484eef1b066e43e55d44e6ffd343f556de31bccb